File: algoritmos/Exercício 2/Exercicio2_UEPG/Classes/pato.js
/*Andorinhas e patos implementam uma interface para o vôo.*/
class Pato extends Ave{
    constructor(peso, patas){
        super(peso, patas);
    }

    nadar() {
        let mensagem = "Sou um pato e estou nadando";
        alert(mensagem);
        return mensagem;
    }
}

File: algoritmos/Algoritmo 2/Algoritmo2/src/main/java/com/fabio/algoritmo2/Pato.java
package com.fabio.algoritmo2;

/**
 *
 * @author fabio
 */

public class Pato extends Ave implements AveVoo {
    
    public Pato(double peso, int n_patas){
        super(peso, n_patas);
    }
    
    @Override
    public void voar(){
        System.out.println("estou voando.\n");
    }
}